code/modules/client/client procs.dm

```diff
@@ -162,6 +162,12 @@
 						else
 							src.DB_species_unlock("Grey",30)
 							return
+					if("7")
```

Comment:
Alright, though I'm not sure if that will mean this 30 kp option will display after the 100 kp plasmamen